vllm/transformers_utils/image_processor.py

```diff
@@ -40,6 +39,3 @@ def get_image_processor(
             raise e
 
     return processor
-
-
-cached_get_image_processor = lru_cache(get_image_processor)
```

Comment:
Why did we move this out of this file to `image.py`? IMO it makes more sense to keep it here.

Reply:
It is to avoid confusion with the existing `cached_get_tokenizer` which has a different meaning.